spec/requests/recipes/recipe_spec.rb

```diff
@@ -195,4 +195,60 @@
       end
     end
   end
+
+  describe 'DELETE destroy' do
+    subject { delete recipe_path(recipe) }
+
+    let!(:user) { create(:user) }
+    let!(:recipe) { create(:recipe, user_id: user.id) }
+
+    context 'when logged in' do
+      before do
+        sign_in user
+      end
+
+      it 'destroys the recipe' do
+        expect {
+          subject
+        }.to change(Recipe, :count).by(-1)
+      end
+
+      it 'redirects to the recipes path with a success notice' do
+        subject
+        expect(response).to redirect_to(recipes_path)
+        expect(flash[:notice]).to eq('Recipe successfully removed.')
+      end
+    end
+
+    context 'when destroy fails' do
+      before do
+        allow_any_instance_of(Recipe).to receive(:destroy).and_return(false)
```

Comment:
Instead of doing this, could we try to destroy a recipe that is not from the user logged in for example, what do you think?